burn-ndarray/Cargo.toml

```diff
@@ -21,18 +21,29 @@ blas-openblas-system = [
     "openblas-src/system",
 ]
 
+# Use randomly generated seed during build. This is useful for testing purposes.
+build-generated-seed = []
+
 [dependencies]
-burn-tensor = { version = "0.5.0", path = "../burn-tensor" }
-blas-src = { version = "0.8.0", default-features = false, optional = true }
-openblas-src = { version = "0.10.5", optional = true }
+blas-src = {version = "0.8.0", default-features = false, optional = true}
```

Comment:
I would have a `no_std` and a `std` feature flag on all the crates that can be built with `no_std`.

Reply:
OK. I'll add `std` flag, which is the recommended way for such situations. Having `std` will bring in extra features (following additive nature of feature flags)